src/Tribe/Views/V2/Customizer.php

```diff
@@ -555,7 +555,7 @@ public function filter_global_elements_css_template( $css_template, $section, $c
 
 			// overrides for tec components/full/_ical-link.pcss.
 			$css_template .= "
-				$tribe_events .tribe-events-c-ical__link {
+				$tribe_events .tribe-events-c-ical__link:not(:hover):not(:focus):not(:active) {
```

Comment:
I would add a `color` style to the `hover`, `focus`, and `active` case below to override this style (I assume the CSS files do this too) rather than tagging every single case that this shouldn't target (increases specificity and if there are any cases we need to add it here too).

Reply:
I did - the issue is that the color has to be hard coded as the custom props aren't available to the Customizer - yet.